### DISCUSSION TITLE
GitHub showing closed issues in open issues list

### DISCUSSION THREAD

**Discussion by outloudvi** [community/community#23779] [New to GitHub]
<p>As said by the title, I was observing a closed issue in open issues list. It seems to be also on the GitHub API, since I also reproduce this on third party GitHub client (such as FastHub). I think it’s likely to be a bug. Since this repository has been renamed before, it could be the rename process that caused this bug.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/831892596d8de4de974a53622bc805eaca64b90f.png" data-download-href="/uploads/short-url/iHJbYFXfASc22yHwj6nRR0JMivR.png?dl=1" title="github-open-issue-show-closed.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19144373/181079302-eeb3cb6f-2e9d-41a3-8d72-6c224365a0d8.png" alt="github-open-issue-show-closed.png" data-base62-sha1="iHJbYFXfASc22yHwj6nRR0JMivR" width="690" height="252" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/831892596d8de4de974a53622bc805eaca64b90f_2_690x252.png, https://aws1.discourse-cdn.com/github/original/1X/831892596d8de4de974a53622bc805eaca64b90f.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/831892596d8de4de974a53622bc805eaca64b90f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/831892596d8de4de974a53622bc805eaca64b90f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github-open-issue-show-closed.png</span><span class="informations">1008×369 77.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <img src="https://user-images.githubusercontent.com/19144373/181079304-8c8bf3fc-cdf1-4079-a50b-9be488f1ef6c.png" alt="github-api-open-issue.png" data-base62-sha1="bzn6LPcNpNRGB65jHcfEPxGVQJG" width="348" height="319">

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/outloudvi">@outloudvi</a>,</p>
<p>Thanks for being here! It appears the issue is resolved now.</p>

**Comment by outloudvi** [comment]
<p>Thanks. I can confirm on my side that it’s fixed.</p>

**Comment by unicolet** [comment]
<p>I’m still seeing this on the github <code>issues/assigned</code> page. Is there anything I have to do perhaps?</p>

**Comment by unicolet** [comment]
<aside class="quote no-group quote-modified" data-username="unicolet" data-post="4" data-topic="3301" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 unicolet:</div>
<blockquote>
<p>I’m still seeing this on my github <code>issues/assigned</code> page. Is there anything I have to do perhaps?</p>
</blockquote>
</aside>
<p>Apologies for the insistence, but I’ve been experiencing this for a while. Thank you :slight_smile:</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/unicolet">@unicolet</a> thanks for reaching out! Would you mind sharing the repo where you see this?</p>

**Comment by unicolet** [comment]
<p><a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a> unfortunately they’re in private repos, is there a way I could share them with you without posting publicly :thinking: ?</p>
<p>thanks for the quick response :slight_smile:</p>

**Comment by AndreaGriffiths11** [comment]
<p>Unfortunately no. Since it’s related to a private repo, the private support folks are the ones who would be able to offer help with this. If you haven’t already, please contact them at <a href="https://github.com/contact">https://github.com/contact</a></p>

**Comment by unicolet** [comment]
<p>thank you <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>! I will reach out there :slight_smile:</p>
